monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -37,7 +37,7 @@ See xref:../../scalability_and_performance/optimizing-storage.adoc#recommended-c
 
 * Dedicate sufficient local persistent storage to ensure that the disk does not become full. How much storage you need depends on the number of pods. For information on system requirements for persistent storage, see xref:../../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#prometheus-database-storage-requirements[Prometheus database storage requirements].
 * Make sure you have a Persistent Volume (PV) ready to be claimed by the Persistent Volume Claim (PVC), one PV for each replica. Since Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack. The Persistent Volumes should be available from the Local Storage Operator. This does not apply if you enable dynamically provisioned storage.
-* Use the block type of storage.
+* Use the filesystem type of storage.
```

Comment:
Should not be changed, monitoring supports block storage
For example, AWS EBS, it is a block storage

Reply:
@juzhao wait, i am confused, afaik @brancz and @paulfantom mentioned we do only support filesystem storage, not block storage.

Reply:
We do support block storage, the crucial thing is that we need a POSIX filesystem, and specifically NFS has shown to be problematic with Prometheus/Alertmanager so that's something we want to explicitly exclude and say we need a POSIX compliant filesystem.

Reply:
I think we should remove mention of the type of storage we need and only specify we don't support NFS. In other cases `volumeMode: filesystem` enforces a filesystem to be present and it is a default setting for most PV providers since k8s 1.13 ([docs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-mode)).

Reply:
I think I understand what you meant by block storage now. Yes we do require a filesystem to be present when mounted into Prometheus, but that filesystem may very well itself be on block storage. I think @paulfantom's suggestion sounds adequate.

Reply:
sounds good, thanks a lot for the clarification :+1: @maximsvistunov are you all set with the information above?

Reply:
Thanks for the clarifications everybody. @s-urbaniak , yes, but please check 2fb0c4e (maybe @paulfantom too), especially the deletion of `volumeMode: filesystem` per Junqi's feedback. If everything is correct, we can merge this.

Reply:
@paulfantom gave the final ACK, I'm requesting to merge this.